packages/core/README.md

```diff
@@ -24,6 +24,11 @@ Examples can be found - [here](https://github.com/kenshoo/react-tree/blob/master
  yarn add @kenshooui/react-tree
```

Comment:
lets ensure the readme files contain a link to the demo page.

Reply:
It will be possible only after this PR will be merged ,no?